src/screens/FeedScreen/Header.js

```diff
@@ -456,21 +456,21 @@ const styles = StyleSheet.create({
   feedDate: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: COLORS.blackgrey,
+    color: colors.blackgrey,
     lineHeight: 18
   },
   feedDateLocation: {
     flex: 1,
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: COLORS.blackgrey,
+    color: colors.blackgrey,
     lineHeight: 18
   },
   point: {
     width: 2,
     height: 2,
     borderRadius: 4,
-    backgroundColor: COLORS.gray8,
+    backgroundColor: colors.gray,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values have been updated to use the new `colors` import. Ensure that these new color values are equivalent to the old ones and that they match the intended design.
```diff
-    color: COLORS.blackgrey,
+    color: colors.blackgrey,

-    color: COLORS.blackgrey,
+    color: colors.blackgrey,

-    backgroundColor: COLORS.gray8,
+    backgroundColor: colors.gray,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->